### PR TITLE
Hw3

### PR DESCRIPTION
**Pull request by BadSolver**
1)Task: https://www.figma.com/file/D3hs2PC8uClFPe6iTTNXat/TMS_front?node-id=617%3A1270
image
3)deadline: 24.02.2022 / done : 19.02.2022
4) deploy: https://badsolver.github.io/TMS/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+                    </div>
+                    <p class="name_customer">Davis Rhiel Madsen <br> <span>Manager</span> </p>
+                    <div class="ava__customer">
+                        <img src="./image/Ellipse8.png" alt="ava">
+                    </div>
+                    <p class="name_customer">Jaydon Rosser <br> <span>Manager</span> </p>
+                    <div class="ava__customer">
+                        <img src="./image/Ellipse9.png" alt="ava">
+                    </div>
+                    <p class="name_customer">Cooper Dorwart <br> <span>Manager</span> </p>
```
Comment (shuliakleonid) [inline index.html]: class customer, naming I don't understand. If should write like customer__element

File: index.html [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+  </section>
+  <section class="city">
+      <div class="wrapper__city">
+            <div class="container__city">
+                <h2 class="city_title">STYLISH AXURE DESIGN</h2>
+                <hr class="hr_city">
+                <p class="about_city">Use the sections you need, remove the ones you don't.  Create gorgeous prototypes faster than ever!</p>
+                <button class="city_btn">Download</button>
+            </div>
+        </div>
```
Comment (shuliakleonid) [inline index.html:239]: same with class city

File: style.css [hunk 3]
```diff
@@ -0,0 +1,592 @@
...
+    width: 1170px;
+    margin-left: 170px;
+}
+.input_ban.act {
+    margin-left: 100px;
+    margin-top: 50px;
+    margin-bottom: 50px;
+}
+.input_ban.act_one {
+
```
Comment (shuliakleonid) [inline style.css]: delete unuse class

**Review by shuliakleonid — CHANGES_REQUESTED** [review]
image
# Edit hover effect

File: index.html (hunk 1 above)
Comment (BadSolver) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (BadSolver) [reply index.html:239]: done

File: style.css (hunk 3 above)
Comment (BadSolver) [reply style.css]: done

**Review by shuliakleonid — APPROVED** [review]

**Review by shuliakleonid — APPROVED** [review]

**Review by KseniaKotova — APPROVED** [review]

**Review by AlexisKozlov — APPROVED** [review]

**Pull request merged**